### PR TITLE
Fix botched break dialog

### LINKED ISSUE #63: Botched Break Dialog
Hi, it's me again with some issues regarding the premium design one is faced with when using pyradium :upside_down_face:.

The nice background shadow on the dialog that opens when a break is started seems to have a vertical gap on the left side of the presentation. This is misleading to anyone looking at the presentation during pauses and needs to be fixed immediately.

Fixing this is as simple as removing the default padding that the lovely browser sets by default for `body` and adding the 10px margin back to the slide's `.size_container`. However, a future-proof approach might be to use the [html dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog). What do you think about this?

### PR DESCRIPTION
**Pull request by tametsi**
Use html `<dialog>` element instead of a custom written dialog whose backdrop does not cover 100% of the UI.
Additionally, some minor design improvements have been added.

Unfortunately, I am not able to further maintain this dialog because of the ugly blue border around the dialog which adds extreme complexity to the construct. Hope this still meets your expectations.

Fixes #63.

### CODE REVIEW

File: pyradium/templates/base/pyradium_dialog.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+dialog {
+	border: none;
+	border-radius: 10px;
+	padding: 20px;
+	border: 15px solid #3498db;
+}
+::backdrop {
```
Comment (johndoe31415) [inline pyradium/templates/base/pyradium_dialog.css:7]: Missing newline

File: pyradium/templates/base/pyradium_dialog.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	background: rgba(0, 0, 0, 0.45);
+}
+
+dialog .header {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	flex-flow: row nowrap;
+}
+dialog .close {
```
Comment (johndoe31415) [inline pyradium/templates/base/pyradium_dialog.css:17]: newline

File: pyradium/templates/base/pyradium_dialog.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	padding: 0;
+	margin: 0;
+	border: none;
+	background-color: inherit;
+	font-weight: bold;
+	width: 48px;
+	height: 48px;
+	opacity: 0.5;
+}
+dialog .close:hover {
```
Comment (johndoe31415) [inline pyradium/templates/base/pyradium_dialog.css:28]: newline

File: pyradium/templates/base/pyradium_dialog.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	opacity: 1;
+}
+dialog .placeholder {
```
Comment (johndoe31415) [inline pyradium/templates/base/pyradium_dialog.css:31]: newline

File: pyradium/templates/base/pyradium_dialog.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+	width: 48px;
+}
+dialog .icon {
```
Comment (johndoe31415) [inline pyradium/templates/base/pyradium_dialog.css:34]: newline

File: pyradium/templates/base/pyradium_dialog.css [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+	display: flex;
+	justify-content: space-between;
+	flex-flow: row nowrap;
+}
+dialog .close {
+	font-size: 32px;
+	padding: 0;
+	margin: 0;
+	border: none;
+	background-color: inherit;
```
Comment (johndoe31415) [inline pyradium/templates/base/pyradium_dialog.css:22]: Is background-color inherit not the default? I'm not sure. If it is, please omit.

**Review by johndoe31415 — CHANGES_REQUESTED** [review]
Thanks for the PR. Review attached. Have not tested it yet, just from looking at the code. Also suggest rewording of the text. Suggested summary line would be "Aesthetic improvements of modal" and in the text, clarify that it uses the HTML (upper case) dialog element instead of a custom div. Additionally, add Closes #64.

**Comment by johndoe31415** [comment]
Also, regarding the color: Suggest "7px solid #aaa" for that. Should look a bit better, no?

**Comment by johndoe31415** [comment]
Just tried it out. The coffee mug looks super nice, I like it!

File: pyradium/templates/base/pyradium_dialog.css (hunk 6 above)
Comment (tametsi) [reply pyradium/templates/base/pyradium_dialog.css:22]: No, because this is a hTmL `<button />` element, some default background color is set by the browser hence this line of code is quite important.

**Comment by tametsi** [comment]
Oops, that must have been the `f` flag...

**Comment by tametsi** [comment]
> Also, regarding the color: Suggest "7px solid #aaa" for that. Should look a bit better, no?

A bit better? Hmmm, maybe. Still ugly? Yeah, would say so... :)

**Comment by johndoe31415** [comment]
Looks great, merged to master as-is. No changes from my side ;-) Thanks for the PR. You will be able to enjoy the fruit of your labor next week!
